src/main/java/net/dv8tion/jda/core/JDA.java

```diff
@@ -156,6 +157,8 @@ public int getShardTotal()
      */
     User getUserById(String id);
 
+    RestAction<User> retrieveUser(String id);
```

Comment:
It should also come after all of the getUser methods, but before the getGuild methods. It should not be placed between them